bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -4397,4 +4397,24 @@
         <comment>Override SQL query for Get Patient Details in Ward</comment>
         <sqlFile path="BahmniBMPatientListInWard.sql"/>
     </changeSet>
+
+    <changeSet id="201905101200" author="Angshu,Bindu">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                select count(*) from scheduler_task_config where schedulable_class= "org.openmrs.module.emrapi.adt.CloseStaleVisitsTask" and name = "Close Stale Visits Task"; </sqlCheck>
+        </preConditions>
+        <comment> Add Bahmni preferred EMR API scheduler which closes stale visits </comment>
+        <insert tableName="scheduler_task_config">
+                <column name="name" value="Close Stale Visits Task" />
+                <column name="description" value="Auto close visit task after number of hours specified by emrapi.visitExpireHours" />
+                <column name="schedulable_class" value="org.openmrs.module.emrapi.adt.CloseStaleVisitsTask" />
+                <column name="start_on_startup" valueBoolean="true" />
+                <column name="start_time_pattern" value="MM/dd/yyyy HH:mm:ss" />
+                <column name="start_time" valueDate="2014-04-30T23:59:59" />
```

Comment:
made the changes

Reply:
actually I think this was intentionally put, because we wanted the scheduler to kick in only at midnight. This is specifically for closing visits and we don't want it kick in at anytime the server starts